copy/pq.go

```diff
@@ -196,7 +195,28 @@ func (cb *CopyWithPq) saveOrLoadKeysetPaginatedTableIdRange(ctx context.Context,
 	return idRangeSet, nil
 }
 
+func getTables(ctx context.Context, txn pgx.Tx) ([]string, error){
+
+	rows, err := txn.Query(ctx, "SELECT table_name FROM information_schema.tables WHERE table_type='BASE TABLE' AND table_schema='public'")
```

Comment:
Looking at this now, we probably need to grab the schema and table name since a DB can have stuff like `shard1.accounts` and `shard2.accounts`. Where `shard1` is the schema, and `accounts` is the table.